inference-engine/ie_bridges/java/tests/CNNNetworkTests.java

```diff
@@ -0,0 +1,46 @@
+import junit.framework.TestCase;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+import org.intel.openvino.*;
+
+public class CNNNetworkTests extends TestCase {
+    ConfigTests config = new ConfigTests();
+    
+    IECore core;
+    protected String exceptionMessage;
+    
+    @Override
+    protected void setUp() {
+        try {
+            System.loadLibrary(IECore.NATIVE_LIBRARY_NAME);
+        } catch (UnsatisfiedLinkError e) {
+            System.err.println("Failed to load Inference Engine library\n" + e);
+            System.exit(1);
+        }
+        core = new IECore();
+    }
+
+    @Override
+    protected void tearDown() {
+        exceptionMessage = "";
+    }
+
+    public void testGetInputsInfo() {
+        CNNNetwork net = core.ReadNetwork(config.getXML());
+        Map<String, InputInfo> inputsInfo = net.getInputsInfo();
+        for (Map.Entry<String, InputInfo> entry : inputsInfo.entrySet())
+            assertTrue(entry.getValue() instanceof InputInfo);
```

Comment:
not sure that this is useful test - let's keep only the next one. Correct me if I'm wrong but I think this condition is true apiori.